src/components/Navigation.js

```diff
@@ -0,0 +1,27 @@
+import { useDispatch } from 'react-redux';
+import { CiSearch } from 'react-icons/ci';
+import { AiOutlineHome } from 'react-icons/ai';
+import { NavLink } from 'react-router-dom';
+import { searchCity } from '../redux/cities/citiesSlice';
+
+export default function Navigation() {
+  const dispatch = useDispatch();
+  return (
+    <div className="nav-bar">
+      <div className="left">
+        <h2 className="logo-text">
+          <span className="logo-ty">C</span>
+          it
+          <span className="logo-ty">y</span>
+        </h2>
+        <NavLink to="/">
+          <AiOutlineHome className="home-icon" />
+        </NavLink>
+      </div>
+      <div className="search-bar">
+        <input className="search" onChange={(event) => dispatch(searchCity(event.target.value))} type="search" placeholder="Search city" />
+        <CiSearch className="search-icon" />
+      </div>
```

Comment:
- [ ] Great job working on the navigation but I think it would be great if you make it the same as the given design so I recommend to change it to be like the given design  since it is very important to stick with the design layout as mentioned in the requirements 🔥🚀✨🙏🏻 kindly update it in both pages in home page and in details page as well ✨🙏🏻

- You can put the search bar in the header 🔥

- Your
![image](https://github.com/BranBayou/random-cities/assets/58553711/d10139e1-d079-4797-9036-cc1925379dd0)

- Given
## Home page nav
![image](https://github.com/BranBayou/random-cities/assets/58553711/96bd64b6-6b98-44dc-9998-9a990984a81a)

## details page nav

![image](https://github.com/BranBayou/random-cities/assets/58553711/ce85585f-c8ed-442d-a607-ec2b63951f30)